### PR TITLE
Jessie adaptions

### PR DESCRIPTION
**Pull request by rubo77**
There are still some workarounds in the [jessie tree of ffnord-example/bootstrap.sh](https://github.com/rubo77/ffnord-example/blob/jessie/bootstrap.sh), but then it already seems to work fine on jessie.

Only the isc-dhcp-server still doesn't stay running after a restart. but this problem already existed before.

### CODE REVIEW

File: files/usr/local/bin/check-gateway [hunk 1]
```diff
@@ -69,3 +69,6 @@ if [ "$NEW_STATE" == "off" ]; then
 fi
 
 # vim: noai:ts=4:sw=4:ff=unix:ft=text:fdm=marker
+echo "conntrack sollte auf 1048576 gesetzt sein:"
```
Comment (sargon) [inline files/usr/local/bin/check-gateway]: This will generate cron mails on every run.

File: templates/etc/sysctl.d/routing.conf [hunk 2]
```diff
@@ -8,8 +8,8 @@ net.ipv6.conf.all.forwarding=1
 net.ipv4.icmp_errors_use_inbound_ifaddr=1
 
 # Decrease tcp timeout
-net.ipv4.netfilter.ip_conntrack_tcp_timeout_established=<%=conntrack_tcp_timeout%>
-net.ipv4.netfilter.ip_conntrack_udp_timeout=<%=conntrack_udp_timeout%>
+net.ipv4.netfilter.ip_conntrack_tcp_timeout_established=<%=@conntrack_tcp_timeout%>
```
Comment (sargon) [inline templates/etc/sysctl.d/routing.conf]: Why do we need a line break here

**Comment by sargon** [comment]
Except for my comments this looks fine for me, plz fix.

**Comment by MTRNord** [comment]
Info Kommentar: Meine PR ( #176 )hatte dadurch, dass ich die lint fehler berichtigt habe auch zeilen verschoben. Deswegen der conflict.

**Pull request merged**